docs/sources/operators-guide/configuring/configuring-custom-trackers.md

```diff
@@ -25,14 +25,37 @@ Each custom tracker counts the active series matching its label pattern on a per
 
 Series with metric name `cortex_ingester_active_series_custom_tracker` have two labels applied: `name` and `user`. The value of the `name` label is the name of the custom tracker specified in the configuration. The value of the `user` label is the tenant-id for which the series count applies.
 
-Assume two custom trackers are configured as in the example above, and that your Grafana Mimir cluster has three tenants: `tenant_1`, `tenant_2`, and `tenant_with_only_prod_metrics`. Assume all series within `tenant_with_only_prod_metrics` have labels that match the pattern `{namespace=~"prod-.*"}` and none that match `{namespace=~"dev-.*"}`.
+To illustrate, assume that two custom trackers are configured as in the yaml shown above, and that your Grafana Mimir cluster has two tenants: `tenant_1` and `tenant_with_only_prod_metrics`. Assume that `tenant_with_only_prod_metrics` has 3 series with labels that match the pattern `{namespace=~"prod-.*"}` and none that match `{namespace=~"dev-.*"}` and that `tenant_1` has 5 series that match the pattern `{namespace=~"dev-.*"}` and 10 series that match the pattern `{namespace=~"prod-.*"}`.
 
-In this example, the following output appears when the `/metrics` endpoint for the ingester component is scraped:
+You should see the following output when the `/metrics` endpoint for the ingester component is scraped:
 
 ```
-cortex_ingester_active_series_custom_tracker{name="dev", user="tenant_1"}
-cortex_ingester_active_series_custom_tracker{name="prod", user="tenant_2"}
-cortex_ingester_active_series_custom_tracker{name="prod", user="tenant_with_only_prod_metrics"}
+cortex_ingester_active_series_custom_tracker{name="dev", user="tenant_1"}                         5
+cortex_ingester_active_series_custom_tracker{name="prod", user="tenant_1"}                       10
+cortex_ingester_active_series_custom_tracker{name="prod", user="tenant_with_only_prod_metrics"}   3
 ```
 
+Starting with Mimir version 2.1 default configuration as described above can be overridden for specific tenants in the [runtime configuration]({{< relref "./about-runtime-configuration.md" >}}).
+
```

Comment:
```suggestion
For specific tenants, you can override the default configuration as previously described. To do so, edit the  [runtime configuration]({{< relref "./about-runtime-configuration.md" >}}).

```
* The documentation is already versioned, so this version number need not be embedded in the content itself. It could make sense to add a version number in a migration guide, if that is needed. Is it?
* Remove directional language because text does not always flow above, below, left, or right; prefer _preceding_ or _that follows_ as an alternative to each.
* Replace passive voice with active voice.
* Simplify the entire sentence by breaking it up into two; there are many details listed in a single sentence that make it too compact to read, especially for non-native speakers.